newsfragments/7633.feature.rst

```diff
@@ -0,0 +1 @@
+Added TOS acceptance on login if the server provides them
```

Comment:
Terms of Services (ToS) can now be defined in the server. Users are requested to accept them on first connection and whenever they are updated.